src/components/chip/index.tsx

```diff
@@ -303,22 +303,30 @@ const Chip = ({
   [avatarProps, badgeProps, iconSource, rightIconSource, onDismiss]);
 
   const getContainerSize = useCallback(() => {
-    const width = useSizeAsMinimum ? 'minWidth' : 'width';
-    const height = useSizeAsMinimum ? 'minHeight' : 'height';
-
-    return typeof size === 'object'
-      ? {[width]: _.get(size, 'width'), [height]: _.get(size, 'height')}
-      : {[width]: size, [height]: size};
+    const width = typeof size === 'object' ? _.get(size, 'width') : size;
+    const height = typeof size === 'object' ? _.get(size, 'height') : size;
+    return useSizeAsMinimum ? {minWidth: width, minHeight: height} : {width, height};
```

Comment:
I would separate the return to a different method so getContainerSize will only return the size, i.e. height and width so you can use it in the hitSlop calc and wont have to extract it from the style object.

Reply:
Yes